zhmcclient/_adapter.py

```diff
@@ -347,3 +347,44 @@ def __repr__(self):
                 _ports=repr_manager(self._ports, indent=2),
             ))
         return ret
+
+    @logged_api_call
+    def change_crypto_type(self, crypto_type, zeroize=None):
+        """
+        Reconfigures a cryptographic adapter to a different crypto type.
+        This operation is only supported for cryptographic adapters.
+
```

Comment:
According to the HMC API bok, the crypto adapter must be varied offline for this operation to succeed. I suggest stating this requirement by adding the following sentence:
 "The cryptographic adapter must be varied offline before its crypto type can be reconfigured."